server/coordinator/procedure/transfer_leader.go

```diff
@@ -0,0 +1,240 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package procedure
+
+import (
+	"context"
+	"sort"
+	"time"
+
+	"github.com/CeresDB/ceresmeta/pkg/log"
+	"go.uber.org/zap"
+
```

Comment:
Sorry that the current linter cannot detect these extra empty lines, shall we remove them?